tests/app/questionnaire/test_rules.py

```diff
@@ -175,6 +176,40 @@ def test_evaluate_rule_less_than_with_number(self):
         self.assertFalse(evaluate_rule(when, 5))
         self.assertFalse(evaluate_rule(when, 7))
 
+    def test_evaluate_rule_checkbox_contains_all(self):
```

Comment:
pytest parameterize is a useful tool on tests like these: https://docs.pytest.org/en/latest/parametrize.html

Reply:
Would be a nice simplification but unfortunately didn't have any luck in getting it set up and working so decided to leave as is for now.